roles/openshift_certificate_expiry/test/test_fakeopensslclasses.py

```diff
@@ -1,82 +1,78 @@
-#!/usr/bin/env python
 '''
  Unit tests for the FakeOpenSSL classes
 '''
 
 import os
-import sys
-import unittest
 import pytest
+import sh
+import sys
```

Comment:
We could add this to the lint checks: https://github.com/PyCQA/flake8-import-order or https://pypi.python.org/pypi/flake8-isort